src/sonic-yang-models/yang-models/sonic-system-aaa.yang

```diff
@@ -32,17 +36,28 @@ module sonic-system-aaa {
                 }
 
                 leaf failthrough {
-                    type boolean;
+                    type stypes:boolean_type;
                     description "When set to true, authentication is attempted on next configured server/local in the list upon failure.";
-                    default false;
+                    default False;
+                }
+
+                leaf fallback {
+                    type stypes:boolean_type;
+                    description "Allow AAA fallback";
+                    default False;
                 }
                 
                 leaf debug {
-                    type boolean;
+                    type stypes:boolean_type;
                     description "Enable/disable AAA debugging";
-                    default false;
+                    default False;
                 }
 
+                leaf trace {
+                    type stypes:boolean_type;
```

Comment:
Is there any control plane component acting on this info ?
How could we prevent this from getting persisted in config_db.json file ?

Reply:
From Gang Lv:
cli commands and config db both have this 'trace' fie